app/main.cjsx

```diff
@@ -47,6 +47,7 @@ routes = <Route handler={App}>
     <DefaultRoute name="user-profile" handler={require './pages/profile/feed'} />
     <Route name="user-profile-private-message" path="message" handler={require './pages/profile/private-message'} />
     <Route name="user-profile-stats" path="stats" handler={require './pages/profile/stats'} />
+    <Route name="moderations" handler={require './talk/moderations'} />
```

Comment:
Just updated to the suggested `#/talk/moderation` url and also corresponding `#/project/owner/name/talk/moderation`. Those pages now only show the moderations for the section instead of an aggregate. Link can be accessed through the moderator controls section